launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -101,14 +114,17 @@ def __exit__(self, exep_type, exep_value, trace):
 class _WaitForTopicsNode(Node):
     """Internal node used for subscribing to a set of topics."""
 
-    def __init__(self, name='test_node', node_context=None):
+    def __init__(self, name='test_node', node_context=None,
+                 max_number_of_messages=None):
```

Comment:
nitpick: similar nitpick here, always use a multiple of 4 spaces and if you wrap one thing, wrap them all (in most cases and this one).

```suggestion
    def __init__(self,
        name='test_node',
        node_context=None,
        max_number_of_messages=None,
    ):
```

We prefer this over the "aligned indent" to minimize future diffs in situations like this:

```diff
@@ -1,3 +1,3 @@
-def old_function_name(param1,
-                      param2):
+def new_longer_function_name(param1,
+                             param2):
     # function code

```

vs

```diff
@@ -1,4 +1,4 @@
-def old_function_name(
+def new_longer_function_name(
     param1,
     param2
 ):

```

And unlike the other nitpick like this one, when wrapping arguments to a function in Python, always "uncuddle" the closing parenthesis and colon to delineate the function prototype and the beginning of the function's code, e.g.:

```python
def func(
    parameters,
    of_function,
    prototype):
    start_function_here = 'confusing to see where one stops and the other begins'
```

vs

```python
def func(
    parameters,
    of_function,
    prototype
):
    start_function_here = 'much clearer where the prototype stops and the function begins'
```

Reply:
Our code styles are here: http://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#python

But unfortunately don't go into as much nuance as this, so these are just suggestions.